diff --git a/drivers/pinctrl/pinmux.c b/drivers/pinctrl/pinmux.c
index bab888fe3f8e..558cb55eb4a7 100644
--- a/drivers/pinctrl/pinmux.c
+++ b/drivers/pinctrl/pinmux.c
@@ -120,8 +120,8 @@ static int pin_request(struct pinctrl_dev *pctldev,
 		goto out;
 	}
 
-	dev_dbg(pctldev->dev, "request pin %d (%s) for %s\n",
-		pin, desc->name, owner);
+	//dev_dbg(pctldev->dev, "request pin %d (%s) for %s\n",
+	//	pin, desc->name, owner);
 
 	if ((!gpio_range || ops->strict) &&
 	    desc->mux_usecount && strcmp(desc->mux_owner, owner)) {
